src/Raven.Client/Constants.cs

```diff
@@ -120,6 +120,17 @@ private Certificates()
             public const int MaxNumberOfCertsWithSameHash = 5;
         }
 
+        public class DatabaseSettings
+        {
+            private DatabaseSettings()
+            {
+            }
+
+            public const string ClientId = "DatabaseSettings/Client";
```

Comment:
Why do we need those anyway?

Reply:
Removed _ClientId_
Used as debug tag in _context.ReadForDisk_ (isn't it ?)